tensorflow/python/ops/array_ops.py

```diff
@@ -2454,6 +2455,88 @@ def _all_dimensions(x):
   return range(0, rank(x))
 
 
+@tf_export("multi_one_hot")
+def multi_one_hot(indices, depth_list, on_values_list=None,
+                  off_values_list=None, name=None):
+  """Creates one-hot-encodings for multiple features (columns in a matrix)
+  and concatenates the resulting encodings
+
+  If `indices` is a matrix (batch) with shape `[batch, features]`, the output
+  shape will be `[batch, sum(depth_list)]`:
+
+  Example
+  =========
+
+  Suppose that
+
+  ```python
+    indices =
+    [1, 2]
```

Comment:
Do you want to make this into actually exectuable code? This code block does say three-backticks-python.

Reply:
Yes, I took the doctest format from the other docstrings, but in fact some of the code in between wasn't executable. I fixed this and reworked the example ;)